util/test/prediff-for-stacktrace

```diff
@@ -8,13 +8,26 @@
 #
 #  export CHPL_SYSTEM_PREDIFF=$CHPL_HOME/util/test/prediff-for-stacktrace
 
-# If the good file has a stack trace, the expected output shoul have it too
+# This skips some specific tests (for example tests that produce binary output)
+case "$1" in
+  # binary output, no newline
+  ( mandelbrot | mandelbrot-complex | mandelbrot-dist | mandelbrot-fancy)
+  exit
+  # text output, no newline - which ones?
+  ( declprintint | fwriteIntFile | fwriteMultipleInts | fwriteIntStdout )
+  exit
+esac
```

Comment:
Is the logic above really necessary? Also, I'd prefer this prediff be written in Python or Perl. It would be better to read the output one line at a time and check for the Stacktrace region. As it is, I think the grepping below might mess up test output. But there's no real reason to change anything at all until we get to a region just after a line that was exclusively "Stacktrace".